### PR TITLE
make upgrade to create txt files

### PR DESCRIPTION
**Pull request by estute**
This is the initial creation of the txt files based on the .in files from django cookiecutter + django-waffle

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Core requirements for using this application
 
 Django>=1.8,<2.0          # Web application framework
```
Comment (jmbowman) [inline requirements/base.in]: We don't support anything below 1.11 anymore; could you update that here and in the cookiecutter?

File: requirements/base.in [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
-
+django-waffle==0.12.0
```
Comment (jmbowman) [inline requirements/base.in]: Maybe add a comment that this is pinned to match edx-platform?

File: requirements/base.in (hunk 1 above)
Comment (estute) [reply requirements/base.in]: Sure thing

File: requirements/base.in (hunk 2 above)
Comment (estute) [reply requirements/base.in]: You got it

**Comment by estute** [comment]
merging after fixing notes

**Pull request merged**